packages/app/server/tls/certs.ts

```diff
@@ -8,6 +8,32 @@ interface CertificateOptions {
   keySize: number;
 }
 
+const ipV6RegExp = /^\[[0-9a-f:]+\]$/i;
+const ipV4RegExp = /^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$/;
+const bytesFromIPv4 = (util as any).bytesFromIPv4;
+const bytesFromIPv6 = (util as any).bytesFromIPv6;
+const bytesToIPv4 = (util as any).bytesToIPv4;
+const bytesToIPv6 = (util as any).bytesToIPv6;
```

Comment:
Is it not possible to use destructuration here ? 

For example:
`const {bytesFromIPv4, bytesFromIPv6, ...} = util as any;`

Reply:
@fbasso Thank you for your comment. I have taken it into account!